### ISSUE TITLE
SI_LULC_pipeline move away from increasing integer sequence? 

### ISSUE THREAD

**Issue by Koesters** [sentinel-hub/eo-learn#74]
Enhancement suggestion.

The SI_LULC_pipeline  example is written expecting an increasing integer sequence [SI_LULC_pipeline] while a real life example in my region has attributes like 

```
#land_cover_val = [0,1,2,3,4,5,6,7,8,9,10]
land_cover_val = [0,1,4,5,7,10,11,12,14,16,18,19,22,23,24,26,28,29,31,32,34,35,36,41,45,51]
```
as inevitably the smaller section has not all landuse classifiers in it. 

A good proportion needs to be rewritten as loads of the code is somehow based on an increasing integer sequence and fails on a list.  

Here is an example of my landuse data.
**https://hlamap.org.uk/data-download/data-download-form**

The Slovenia link is dead. 
 **http://rkg.gov.si/GERK/documents/RABA_2018_10_31.RAR**

linked from 
**https://eo-learn.readthedocs.io/en/latest/examples/land-cover-map/SI_LULC_pipeline.html**

**Comment by devisperessutti** [comment]
Thanks for the suggestions. 

We have also noticed that the filename has been changed. You can get the file on [http://rkg.gov.si/GERK](http://rkg.gov.si/GERK) selecting the `Grafični podatki RABA za celo Slovenijo (shape.rar ~ 500 MB) KoordSistem: D96/TM` file. We'll update the notebooks accordingly.

Which parts of [this](https://github.com/sentinel-hub/eo-learn/blob/master/examples/land-cover-map/SI_LULC_pipeline.ipynb) workflow wouldn't work with your given labels? 

Any more details would help us improve the example.

**Comment by Koesters** [comment]
It's a suggestion and not a bug. I was just interested in doing this in my area. All works fine on your examples. :) 

I tried to find an area that is suitably representative of the other patches. It's in the inner City of Glasgow. CRS.UTM_30N

[27]: 
`class_names = [entry.class_name for entry in LULC]


print('             Class              =  F1  | Recall | Precision')
print('         --------------------------------------------------')
for idx, lulctype in enumerate([class_names[idx] for idx in class_labels]):
    print('         * {0:20s} = {1:2.1f} |  {2:2.1f}  | {3:2.1f}'.format(lulctype,
                                                                         f1_scores[idx] * 100,
                                                                         recall[idx] * 100,
                                                                         precision[idx] * 100))'


I wrote my own adhoc code. 

 I couldn't test if all training labels where in the other patches as the patch size  is quite small. 

Labels and plabels then don't mix. So the confusion matrix goes AWOL showing empty rows. 

[29]:
`classes=[name for idx, name in enumerate(class_names) if idx in class_labels]`

I just thought it might save others some bother if they try to play around with their own neck of woods. :)
